voting-contract/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
hm, should we have one license file for the whole repo instead of per each project?
also you can use https://github.com/near/near-api-js/blob/master/LICENSE and https://github.com/near/near-api-js/blob/master/LICENSE-APACHE for dual licensing.